### PR TITLE
🌟 feat: add datepicker

### PR DESCRIPTION
**Pull request by wingkwong**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] i18n translation has been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ..., provide screenshots if necessary)
feature

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:
add datepicker

### CODE REVIEW

File: src/utils/search.js [hunk 1]
```diff
@@ -65,14 +89,34 @@ export const filterSearchOptions = (options, text, size) =>
     ).slice(0, size),
   }))
 
+export const filterByDate = node => {
+  console.log(node)
```
Comment (nandiheath) [inline src/utils/search.js]: console.log

File: src/pages/high-risk.js [hunk 2]
```diff
@@ -295,9 +320,18 @@ const HighRiskPage = ({ data, pageContext }) => {
                   JSON.stringify(historiesToSave)
                 )
               }
+              console.log(selectedArray)
```
Comment (nandiheath) [inline src/pages/high-risk.js]: console.log

**Review by nandiheath — CHANGES_REQUESTED** [review]

**Comment by cswbrian** [comment]
There's some z-index issue between search and datepicker component
image

**Comment by cswbrian** [comment]
The datepicker should filter any cards which overlap the selected range,

i.e. if I select 2020-02-01 -> 2020-02-15
the card of 2020-01-30 - 2020-02-14 should also appear

**Comment by wingkwong** [comment]
> The datepicker should filter any cards which overlap the selected range,
> 
> i.e. if I select 2020-02-01 -> 2020-02-15
> the card of 2020-01-30 - 2020-02-14 should also appear

image

**Comment by cswbrian** [comment]
The close button of datepicker is not working in desktop view.
Can you also change the date format to DD/MM/YYYY?
image

**Comment by wingkwong** [comment]
> The close button of datepicker is not working in desktop view.
> Can you also change the date format to DD/MM/YYYY?
> image

@cswbrian 
1. I think it was due to the z-index fix. However, the z-index issue cannot be reproduced now on my side after rollbacking that change. The behaviour I noticed is that the datepicker will be closed if I click on AsyncSelect. Not sure why.

2. Added ``displayFormat``

**Pull request merged**